apps/recovery-portal/src/main/webapp/challenge-question-add.jsp

```diff
@@ -33,8 +34,9 @@
     String errorMsg = IdentityManagementEndpointUtil.getStringValue(request.getAttribute("errorMsg"));
 
     Map<String, List<ChallengeQuestionDTO>> challengeQuestionSets = new HashMap<String, List<ChallengeQuestionDTO>>();
+    String username = session.getAttribute("username");
 
-    if (session.getAttribute("username") != null) {
+    if (!StringUtils.isBlank(username)) {
```

Comment:
Can we use `StringUtils.isNotBlank()` method? Check other places also.

Reply:
This has been fixed now.

Reply:
Ref https://github.com/yathindrakodithuwakku/identity-apps/blob/6d819794409a7af3c175c492d6d42cc05b3d78f6/apps/recovery-portal/src/main/webapp/challenge-question-add.jsp#L39